test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/BatchTests.cs

```diff
@@ -169,5 +170,70 @@ private void ValidateOperationOutcome(string actualStatusCode, OperationOutcome
             Assert.Equal(expectedIssueType, issue.Code);
             Assert.Equal(expectedDiagnostics, issue.Diagnostics);
         }
+
+        [Theory]
+        [InlineData("true", "true")]
+        [InlineData("true", "false")]
+        [InlineData("false", "true")]
+        [InlineData("false", "false")]
+        public async Task GivenABundle_WithProfileValidationFlag_ReturnsABundleResponse(string headerValue, string profileValidation)
+        {
+            var bundle = new Hl7.Fhir.Model.Bundle
+            {
+                Type = Bundle.BundleType.Batch,
```

Comment:
We should replicate this test for SQL / Transaction bundles. Difference in behavior should be if one request fails the bundle will fail

Reply:
I replicated and added similar test for Transaction Bundle.